pkg/scheduler/plugins/core/availabilitynodepriority/availability_node_priority.go

```diff
@@ -74,11 +73,6 @@ func (pl *AvailabilityNodePriority) Score(ctx context.Context, args interface{},
 	if states.Replicas > 0 { //need at least a pod to compute spread
 		var skew int32
 
-		//Need to check if there is at least one pod in every node to satisfy HA
-		if !state.SatisfyNodeAvailability(states.SchedulablePods, states) {
-			return 0, state.NewStatus(state.Unschedulable, ErrReasonNotEnoughPods)
-		}
```

Comment:
Can we add a regression test?

Reply:
Yes - thanks! Added tests in 07ecc1f9da758883b5af8a91a2f5bda51a8692b2